body_buffer.go

```diff
@@ -23,7 +23,6 @@ import (
 // BodyBuffer is used to read RequestBody and ResponseBody objects
 // It will handle memory usage for buffering and processing
 type BodyBuffer struct {
-	io.Writer   // OK?
```

Comment:
It was left for developers to know that BodyBuffer implements io.Writer, so they can
```go
io.Copy(bodybuffer, someReader)
```
What do you think @fzipi ?

Reply:
Approved, please add to the documentation that it implements io.Writer